OSXvnc-server/AVSampleBufferHolder.h

```diff
@@ -0,0 +1,21 @@
+//
+//  AVSampleBufferHolder.h
+//  OSXvnc
+//
+//  Created by Mykola Mokhnach on 19.01.19.
+//  Copyright (c) 2019 Sauce Labs Inc. All rights reserved.
```

Comment:
OSXvnc uses GPL 3. This copyright statement violates that license. Could you please fix it?